test/kernel.jl

```diff
@@ -97,4 +100,11 @@
     test_kernel(:collinear, Xc([:gga_c_pbe]), test_compute=false)
     test_kernel(:collinear, Xc([:gga_x_pbe]), test_compute=false)
     test_kernel(:collinear, Xc([:gga_x_pbe, :gga_c_pbe]), test_compute=false)
+
+    @testset "Non-linear core correction (NLCC)" begin
+        psp = TestCases.silicon.psp_upf  # PseudoDojo v0.4.1 Si includes NLCC
```

Comment:
@test psp.has_nlcc (or whatever it's called) just to make sure this does not break randomly in the future ?

Also do we need all three of these tests ? Should not one be sufficient ?

Reply:
Makes sense, I added a test for `has_core_density(psp)` here, and just kept just LDA.